modules/iqtree/main.nf

```diff
@@ -11,11 +11,11 @@ process IQTREE {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
 
-    conda (params.enable_conda ? "bioconda::iqtree=2.1.2" : null)
+    conda (params.enable_conda ? 'bioconda::iqtree=2.1.4_beta' : null)
     if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/iqtree:2.1.2--h56fc30b_0"
+        container "https://depot.galaxyproject.org/singularity/iqtree:2.1.4_beta--hdcc8f71_0"
```

Comment:
For the naming this one seems to be a beta, do we want to update it?

Reply:
Tests seem to be passing 🤷🏽 It can be downgraded manually later if required.